### PR TITLE
Add support for 2.9 and minor updates

### PR DESCRIPTION
**Pull request by dgarros**
This PR add the support for NetBox 2.9 and add 2.9-beta2 for testing in travis.

Also as part of this PR:
- Fixed an issue when reports have no data
- Add the tag for module name in the metrics for reports
- Update small description in the readme
- Add a simple report as an example

### CODE REVIEW

File: development/example_reports/users.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+class CheckUser(Report):
+    """Report for Users."""
+
+    def test_is_uppercase(self):
+        """Check that every user has his/her name in lowercase."""
+        for user in User.objects.all():
+
+            # Skip if not master of virtual chassis as only master should have primary IP
```
Comment (glennmatthews) [inline development/example_reports/users.py:13]: Comment seems out of place.

File: netbox_metrics_ext/metrics.py [hunk 2]
```diff
@@ -48,12 +51,25 @@ def metric_reports():
         Iterator[GaugeMetricFamily]
             netbox_report_stats: with report name and status as labels
```
Comment (glennmatthews) [inline netbox_metrics_ext/metrics.py:52]: Looks like this docstring should be updated as you're adding a third label.

File: development/example_reports/users.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+"""This script contains an example of report about users."""
+from extras.reports import Report
+from django.contrib.auth.models import User
```
Comment (glennmatthews) [inline development/example_reports/users.py:3]: To make pylint-django happier, instead do:

```python
from django.contrib.auth import get_user_model
User = get_user_model()
```

File: netbox_metrics_ext/metrics.py (hunk 2 above)
Comment (dgarros) [reply netbox_metrics_ext/metrics.py]: updated

**Comment by dgarros** [comment]
Thanks @glennmatthews for the review and for providing most of the code for this PR. it's been super helpful

**Pull request merged**